packages/dev/test/NetworkSimultaion.test.ts

```diff
@@ -161,7 +154,48 @@ contract('Network Simulation for Relay Server', function (accounts) {
       })
     })
 
-    async function sendMultipleRelayedTransactions (): Promise<void> {
+    describe('boosting & rebroadcasting all transactions', function () {
+      let id: string
+      before(async () => {
+        id = (await snapshot()).result
+      })
+      after(async () => {
+        await revert(id)
+      })
+      it('should boost underpriced transactions and only rebroadcast fairly priced transactions', assertBoostAndRebroadcast)
+      it('should not boost any transactions if latest on-chain nonce is lower than oldest pending tx nonce', async function () {
+        const storedTxs = await env.relayServer.txStoreManager.getAll()
+        const latestNonce = await env.web3.eth.getTransactionCount(storedTxs[0].from)
+        assert.equal(storedTxs[0].nonce, latestNonce)
+        await env.relayServer.txStoreManager.removeTxsUntilNonce(storedTxs[0].from, latestNonce)
+        await evmMineMany(pendingTransactionTimeoutBlocks)
+        try {
+          await env.relayServer._boostStuckPendingTransactions(await env.web3.eth.getBlockNumber())
+          assert.fail()
```

Comment:
I used to write tests like that before I discovered the syntax we use now:
```await expect(promise).to.be.eventually.rejectedWith('Failed to sign relayed transaction for 0x4cfb3f70bf6a80397c2e634e5bdd85bc0bb189ee')```

Reply:
I don't like that syntax... chai annoys me

Reply:
fixed, though I don't like chai in general..

Reply:
> Fixed, though I don't like chai in general..

Nobody does, but what's the alternative?